sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/reporting/AllureSprimber.java

```diff
@@ -175,39 +175,69 @@ private void doWithTestFinish(Node node) {
     }
 
     private List<Label> processTestLabels(Node node) {
-        return node.getAttribute("meta").map(o -> {
+        List<Label> labelList = new ArrayList<>();
```

Comment:
Why do you need to define it here, if using only in the method end

Reply:
just to start with declaring everything, to be clear and honest